rules/linux/execution_linux_netcat_network_connection.toml

```
@@ -24,6 +24,59 @@ index = ["auditbeat-*", "logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Netcat Network Activity"
+note = """## Triage and analysis
+
+### Investigating Netcat Network Activity
+
+Netcat is a dual-use command line tool that can be used for various purposes, such as port scanning, file transfers, and
+connection tests. Attackers can abuse it's functionality for malicious purposes such create bind or reverse shells to
+gain access to the target system.
+
+A reverse shell is a mechanism that's abused to connect back to an attacker-controlled system. It effectively redirects
+the system's input and output and delivers a fully functional remote shell to the attacker. Even private systems are
+vulnerable since the connection is outgoing.
+
+A bind shell is a type of backdoor that the attackers set up on the target host and binds to a specific port to listens
+for an incoming connection from the attacker.
+
+This rule identifies potential reverse or bind shell activity using Netcat by checking for the execution of Netcat
```

Same thing here, I think using the full name for each concept will help each one be understood better:
```suggestion
This rule identifies potential reverse shell or bind shell activity using Netcat by checking for the execution of Netcat
```